.github/workflows/protoc_diff_check.yml

```diff
@@ -20,17 +20,11 @@ permissions:
 
 on:
   push:
-    paths:
-      - 'api/proto/**'
-      - 'docs/openapi/**'
-      - 'pkg/generated/protobuf/**'
-      - 'protoc-builder/**'
+    branches:
```

Comment:
Do we need on push? Making this a required workflow on pull_request could be sufficient

Reply:
I guess not, maybe just main

Reply:
I think we might need it, just in case if we allow merging without the PR content already being in sync with main.